### DISCUSSION TITLE
autofocus?

### DISCUSSION THREAD

**Discussion by e-lu6a** [megahertz/react-simple-wysiwyg#58] [Q&A] [answered]
hello! 

has autofocus been implemented? i noticed that there was a very recent commit for it, but in the demo, and in my own project, it doesn't appear to be working. i'm not sure if i've misunderstood how autofocus should work - in the demo, upon page load, it does not look like the editor is focused. if i click near or within the editor, then it is "focused" - ie. there is a slight style change in the editor, and the blinking cursor appears. 

also, really appreciate how simple and easy to use this package is, as well as its documentation and examples. 
thank you!

**Comment by megahertz** [comment, marked as answer]
Hi. Do you mean this demo: https://megahertz.github.io/react-simple-wysiwyg/? It doesn't use autofocus.

This feature is still new, and I have tested it in only a few cases. First, make sure you use the latest v3.2.0. If it doesn't work, please send a code sample.

**Reply by e-lu6a** [reply]
ahh, thank you! that is embarrassing. it works great!
